server/test/footyPoints.test.js

```diff
@@ -37,7 +37,8 @@ afterAll(async () => {
         const collection = mongoose.connection.collections[collectionName];
         await collection.deleteMany();
     }
-    mongoose.connection.close();
+    await mongoose.connection.close();
```

Comment:
@sbetti02 This operation was async so the test was holding up jest's shutdown hook